### ISSUE TITLE
Application taskbar icon is unresponsive

### ISSUE THREAD

**Issue by netnazgul** [OpenRA/OpenRA#15422]
Running latest playtest on Win10, application icon in taskbar doesn't respond to clicks right away and requires several clicks/seconds to minimize/restore the application window.

https://cdn.discordapp.com/attachments/279290580201373696/473193243111325699/2018-07-29_19-07-44.mp4
https://cdn.discordapp.com/attachments/279290580201373696/473194806928080898/2018-07-10_23-30-00-New.mp4

https://github.com/OpenRA/OpenRA/blob/bleed/OpenRA.Platforms.Default/Sdl2PlatformWindow.cs#L190

Reverting the threading fixes the issue:
```
			context = new Sdl2GraphicsContext(this);
			((Sdl2GraphicsContext)context).InitializeOpenGL();
```

**Comment by pchote** [comment]
I spent some time looking into this, and as far as I can tell the bug is somewhere between Windows and SDL2.  Windows simply isn't processing the events to restore the window.

I can't find any leads on what could be causing this or how to fix it by debugging our code and reading the SDL source.  Setting up a Windows toolchain to compile and debug SDL goes beyond the limits of my time and interest, so I have to give up on this.  My understanding is that @chrisforbes reached the same conclusion.

I'll file a PR that disables the threaded renderer on Windows the same way that we disabled HiDPI support (#13127), and we can try asking for help in the next playtest news post.
